resources/windows/wix/project.wxs.erb

```diff
@@ -44,33 +44,5 @@
       </Component>
     </Directory>
 
-    <SetProperty Id="ARPINSTALLLOCATION" Value="[INSTALLDIR]" After="CostFinalize" />
-    <Property Id="INSTALLDIR">
```

Comment:
Should this removal be a part of this PR? I know it started as a reset of the custom actions work, but it's turned into a revamp of how we're dealing with wix files, both supplied by vanagon and by the proejct